src/service/CinemaService.java

```diff
@@ -0,0 +1,179 @@
+package service;
+
+import exception.AlreadyTakenSeatException;
+import exception.NonExistingSeatException;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class CinemaService {
+
+    private final Scanner scanner;
+    private final Cinema cinema;
+
+    public CinemaService() {
+        scanner = new Scanner(System.in);
+        cinema = initializeCinema();
+        createSeats();
+    }
+
+    private Cinema initializeCinema() {
+        System.out.println("Enter the number of rows:");
+        int rows = scanner.nextInt();
+
+        System.out.println("Enter the number of seats in each row:");
+        int cols = scanner.nextInt();
+
+        System.out.println();
+
+        return new Cinema(rows, cols);
+    }
+
+    private void createSeats() {
+        final String S = "S";
```

Comment:
That's not a constant. Can you please use a more meaningful name than **S**?

Reply:
Fixed